examples/scene/prefab.rs

```diff
@@ -0,0 +1,123 @@
+use bevy::{ecs::schedule::ShouldRun, prelude::*, reflect::TypeRegistry, scene::SceneSpawnError, utils::HashMap};
+/// Basic Prefab support for Bevy 0.5.
```

Comment:
```suggestion
/// Basic Prefab support
```

This comment will quickly go stale in silent ways, so it's best to leave out the version number.